components/support/base/src/main/java/mozilla/components/support/base/http/HttpHeaders.kt

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.support.base.http
+
+/**
+ * A collection of common HTTP header definitions.
+ *
+ * A list of common HTTP request headers can be found at
+ *   https://en.wikipedia.org/wiki/List_of_HTTP_header_fields#Standard_request_fields
+ *
+ * A list of common HTTP response headers can be found at
+ *   https://en.wikipedia.org/wiki/List_of_HTTP_header_fields#Standard_response_fields
+ */
+object HttpHeaders {
+
+    const val CONTENT_TYPE = "Content-Type"
+}
```

Comment:
I like the idea of those constants. But wouldn't we want to move them into `concept-fetch`?

Reply:
I discussed this (briefly) in the issue: https://github.com/mozilla-mobile/android-components/issues/2361#issue-421299227